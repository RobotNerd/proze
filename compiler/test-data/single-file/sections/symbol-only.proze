Title: My Document
Author: Jon Document


Chapter: Example

This is the text of the first section.

---

This is the text of the next section.


Chapter: Another Example

This chapter does not have any sections.
