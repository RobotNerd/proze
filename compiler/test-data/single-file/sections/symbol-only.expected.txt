My Document
by Jon Document


Example

This is the text of the first section.

---

This is the text of the next section.


Another Example

This chapter does not have any sections.
